README.md

```diff
@@ -2,391 +1,312 @@
-    public static void main (String[] args) throws Exception
-    {
-        try {
-            Init init;
-            String consumerSecret = "74c5fd430cf1242a527f6223aebd42d30464be22";
-
-            String secString = "{\"consumer_key\":\"yis0TYCu7U9V4o7M\","
-                    +    "\"domain\": \"assess.vg.learnosity.com\","
-                    +    "\"user_id\": \"$ANONYMIZED_USER_ID\"}";
-
-            String reqString = "{\"state\":\"initial\","
-                    +  "\"type\":\"local_practice\","
-                    +  "\"response_id\":\"60005\","
-                    +  "\"questions\":"
-                    +   "[{\"stimulus_list\":"
-                    +     "[\"London\","
-                    +        "\"Dublin\","
-                    +        "\"Paris\","
-                    +      "\"Sydney\"],"
-                    + "\"stimulus\":\"Match the cities to the parent nation\","
-                    +   "\"type\":\"association\","
-                    +   "\"possible_responses\":"
-                    +   "[\"Australia\","
-                    +     "\"France\","
-                    +     "\"Ireland\","
-                    +       "\"England\"],"
-                    +   "\"validation\":"
-                    +   "{\"valid_responses\":"
-                    +       "[\"England\","
-                    +             "\"Ireland\","
-                    +       "\"France\","
-                    +       "\"Australia\"]}}]}";
-            init = new Init("questions", secString, consumerSecret, new JSONObject(reqString));
-            String questionJson = init.generate();
-            System.out.println(questionJson);
-
-            // Now you can pass questionJson to the initialisation of Questions API
-            // For instance on jsp you might have
-            // <script src="//questions.learnosity.com"></script>
-            // var questionsApp = LearnosityApp.init(questionJson);
-        }catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
-    }
-}
-```
+### **Development install from a Git clone**
+To install from the terminal, run this command:
 
-#### Arguments
-**service**<br>
-A string representing the Learnosity service (API) you want to integrate with. Valid options are:
+    git clone git@github.com:Learnosity/learnosity-sdk-Java.git
 
-* assess
-* author
-* data
-* items
-* questions
-* reports
+### **Alternative method 1: download the zip file**
+Download the latest version of the SDK as a self-contained ZIP file from the [GitHub Releases](https://github.com/Learnosity/learnosity-sdk-Java/releases) page. The distribution ZIP file contains all the necessary dependencies.
 
-**security**<br>
-An Object that includes your *consumer_key* but does not include your *secret*. The SDK sets defaults for you, but valid options are:
+[(Back to top)](#table-of-contents)
 
-* consumer_key
-* domain (optional)
-* timestamp (optional)
-* user_id (optional)
+## Quick start guide
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
## Quick start guide

```